stacks_queues/stack.py

```diff
@@ -12,27 +12,35 @@ def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
         """
-        pass
-
+        self.store.add_first(element)
+       
     def pop(self):
         """ Removes an element from the top
             Of the Stack
             Raises a StackEmptyException if
             The Stack is empty.
             returns None
         """
-        pass
+    
+        return self.store.remove_first()
+       
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        return self.store.empty()
 
     def __str__(self):
```

Comment:
✨ This works, but you might also consider taking advantage of `LinkedList`'s `str()` method